From 071c5966a0d5547ea8e533ae670e0d54d6b03dcf Mon Sep 17 00:00:00 2001
From: Johannes Lorenz <j.git@lorenz-ho.me>
Date: Sun, 25 Oct 2020 14:15:37 +0100
Subject: [PATCH 1637/1936] main.cpp: Don't show LASH/NSM info if not compiled

These info message confuses the user: They make it look as if zyn would
do something with LASH/NSM (may it even be just checking if they are in
use). Not showing the message makes it clear to the user that LASH/NSM
are not compiled into zyn.
---
 src/main.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/main.cpp b/src/main.cpp
index c0ea35af..1d2c3ed0 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -663,7 +663,6 @@ int main(int argc, char *argv[])
             GUI::raiseUi(gui, "/alert-reload", "i", old_save);
         middleware->enableAutoSave(auto_save_interval);
     }
-    printf("[INFO] NSM Stuff\n");
 
     //TODO move this stuff into Cmake
 #if USE_NSM && defined(WIN32)
@@ -677,6 +676,7 @@ int main(int argc, char *argv[])
 #endif
 
 #if USE_NSM
+    printf("[INFO] NSM Stuff\n");
     char *nsm_url = getenv("NSM_URL");
 
     if(nsm_url) {
@@ -691,8 +691,8 @@ int main(int argc, char *argv[])
     }
 #endif
 
-    printf("[INFO] LASH Stuff\n");
 #if USE_NSM
+    printf("[INFO] LASH Stuff\n");
     if(!nsm)
 #endif
     {
@@ -799,7 +799,7 @@ int main(int argc, char *argv[])
 done:
 #endif
         GUI::tickUi(gui);
-#endif
+#endif // !WIN32
         middleware->tick();
 #ifdef WIN32
         Sleep(1);
@@ -817,7 +817,7 @@ done:
         }
 #endif
 #endif
-    }
+    } // while !Pexitprogram
 
     mem_locker.unlock();
 
-- 
2.47.0

